assets/src/dashboard/app/views/myStories/content/storiesView/index.js

```diff
@@ -122,56 +122,62 @@ function StoriesView({
 
-  const handleMenuItemSelected = useCallback(
-    (sender, story) => {
-      setContextMenuId(-1);
-      switch (sender.value) {
-        case STORY_CONTEXT_MENU_ACTIONS.OPEN_IN_EDITOR:
-          trackEvent('open_in_editor');
-          break;
-        case STORY_CONTEXT_MENU_ACTIONS.RENAME:
-          setTitleRenameId(story.id);
-          break;
+  // menu item actions
+  const handleOpenStoryInEditor = useCallback(() => {
+    setContextMenuId(-1);
+    trackEvent('open_in_editor');
+  }, []);
 
-        case STORY_CONTEXT_MENU_ACTIONS.DUPLICATE:
-          trackEvent('duplicate_story');
-          storyActions.duplicateStory(story);
-          break;
+  const handleRenameStory = useCallback((story) => {
+    setContextMenuId(-1);
+    setTitleRenameId(story.id);
+  }, []);
 
-        case STORY_CONTEXT_MENU_ACTIONS.CREATE_TEMPLATE:
-          storyActions.createTemplateFromStory(story);
-          break;
+  const handleDuplicateStory = useCallback(
+    (story) => {
+      setContextMenuId(-1);
+      trackEvent('duplicate_story');
+      storyActions.duplicateStory(story);
+    },
+    [storyActions]
+  );
 
-        case STORY_CONTEXT_MENU_ACTIONS.DELETE:
-          setActiveStory(story);
-          setActiveDialog(ACTIVE_DIALOG_DELETE_STORY);
-          break;
+  const handleCreateTemplateFromStory = useCallback(
+    (story) => {
+      setContextMenuId(-1);
+      storyActions.createTemplateFromStory(story);
+    },
+    [storyActions]
+  );
 
-        case STORY_CONTEXT_MENU_ACTIONS.COPY_STORY_LINK:
-          global.navigator.clipboard.writeText(story.link);
+  const handleDeleteStory = useCallback((story) => {
+    setContextMenuId(-1);
+    setActiveStory(story);
+    setActiveDialog(ACTIVE_DIALOG_DELETE_STORY);
+  }, []);
 
-          addSnackbarMessage({
-            message:
-              story.title.length > 0
-                ? sprintf(
-                    /* translators: %s: story title. */
-                    __('%s has been copied to your clipboard.', 'web-stories'),
-                    story.title
-                  )
-                : __(
-                    '(no title) has been copied to your clipboard.',
-                    'web-stories'
-                  ),
-            severity: ALERT_SEVERITY.SUCCESS,
-            id: Date.now(),
-          });
-          break;
+  const handleCopyStoryLink = useCallback(
+    (story) => {
+      setContextMenuId(-1);
+      global.navigator.clipboard.writeText(story.link);
 
-        default:
-          break;
-      }
+      addSnackbarMessage({
+        message:
+          story.title.length > 0
+            ? sprintf(
+                /* translators: %s: story title. */
+                __('%s has been copied to your clipboard.', 'web-stories'),
+                story.title
+              )
+            : __(
+                '(no title) has been copied to your clipboard.',
+                'web-stories'
+              ),
+        severity: ALERT_SEVERITY.SUCCESS,
```

Comment:
nit: this (`severity: ALERT_SEVERITY.SUCCESS,`) doesn't do anything anymore, it can get deleted. I must have missed it when i gutted the old snackbar

Reply:
moved to #6516